tests/src/test/java/alluxio/worker/block/meta/SpecificTierWriteIntegrationTest.java

```diff
@@ -113,6 +116,9 @@ private void writeFileAndCheckUsage(int writeTier, long memBytes, long ssdBytes,
     Assert.assertEquals("HDD tier usage", hddBytes, bytesOnTiers.get("HDD").longValue());
   }
 
+  /**
+   * Delete all files and related blocks from all tiers.
```

Comment:
`deleteAllFiles` already implies that all tiers and blocks are deleted, so I don't think we need this comment